src/Tizen.NUI/src/public/BaseComponents/ViewAccessibilityEnum.cs

```diff
@@ -37,17 +38,17 @@ public enum AccessibilityInterface
         /// Accessibility interface which can store numeric value
         /// </summary>
         [EditorBrowsable(EditorBrowsableState.Never)]
-        Value = 1,
+        Value = 26,
         /// <summary>
         /// Accessibility interface which can store editable texts
         /// </summary>
         [EditorBrowsable(EditorBrowsableState.Never)]
-        EditableText = 2,
+        EditableText = 9,
         /// <summary>
         /// Accessibility interface which can store a set of selected items
         /// </summary>
         [EditorBrowsable(EditorBrowsableState.Never)]
-        Selection = 3,
+        Selection = 21,
```

Comment:
Could you explain the reason that these numeric values are changed?

Reply:
The value is following

AtspiInterface enum of dali-adaptor/dali/devel-api/adaptor-framework/accessibility.h

I am not sure if there is another way.

Reply:
Correct, these values have to match because they are used to construct a bitmask that is interpreted as `EnumBitSet<AtspiInterface>` in DALi.